problem.py

```diff
@@ -87,7 +87,7 @@ def train_submission(self, module_path, X, y, train_idx=None):
         submission_module = rw.utils.importing.import_module_from_source(
             os.path.join(module_path, self.filename),
             os.path.splitext(self.filename)[0],
-            sanitize=True
+            sanitize=False
```

Comment:
No, it is a mistake from my part. We can keep "sanitize=True" as in the current version since  "pd.read_pickle" is authorized and allows for reading .pkl files.